packages/astro/src/@types/astro.ts

```diff
@@ -95,12 +175,29 @@ export interface AstroGlobalPartial {
 	/**
 	 * @deprecated since version 0.24. See the {@link https://astro.build/deprecated/resolve upgrade guide} for more details.
 	 */
-	resolve: (path: string) => string;
-	/** @deprecated Use `Astro.glob()` instead. */
+	resolve(path: string): string;
```

Comment:
As far as I know, in an interface there's no functional difference between defining a method using an arrow function or like this. 

However, using an arrow function does change JSDoc behavior as we want the JSDoc comment on the function and not the `resolve` attribute, othewise TypeScript won't show the comment (and not get tags such as @deprecated) when using the method

This is particularly relevant here because `Astro.resolve` is deprecated and as such, should be striked-through in editors (which it didn't before this change)

Reply:
Oh! Very interesting, good to know.